src/pages/EventPage/EventPage.tsx

```diff
@@ -130,12 +132,17 @@ function EventPage({ eventId, navigate, ...rest }: EventPageProps) {
                   </>
                 }
                 right={
-                  participants && (
-                    <ParticipantList
-                      participants={participants}
-                      collapseAt={3}
-                    />
-                  )
+                  <>
+                    {resourceLinks && resourceLinks.length !== 0 && (
```

Comment:
Not sure if this is readable at all, but you might be able to shorten this boolean expression into:
```suggestion
                    {esourceLinks?.length && (
```

Reply:
This has been fixed according to @joakimbjerknes